src/parser/monk/mistweaver/modules/features/Abilities.js

```diff
@@ -93,6 +93,16 @@ class Abilities extends CoreAbilities {
         },
         timelineSortIndex: 18,
       },
+      {
+        spell: SPELLS.INVOKE_YULON_THE_JADE_SERPENT,
```

Comment:
Chi-ji is in the talents section. I put Yu'lon here because it is a base Mistweaver ability.

Reply:
adding something to the common/SPELLS/talents files doesnt add it to the spellbook. So unless Chiji is somewhere else in the Abilities file, it would need to be added. I looked through and didnt see it in abilities.js. I believe that is what abe is getting at.